pkg/validation/app_test.go

```diff
@@ -637,7 +687,7 @@ func Test_ValidateApp(t *testing.T) {
 			expectedErr: "validation error: name is not specified for kubeconfig secret",
 		},
 		{
-			name: "case 20: skip validation when giantswarm.io/managed-by label equals flux",
+			name: "case 21: skip validation when giantswarm.io/managed-by label equals flux",
```

Comment:
Hardcore nitpick where I am also interested in opinion of others:
I think it would be nicer to use the index for the case number instead of hardcoding it. So use the index here: https://github.com/giantswarm/app/pull/182/files#diff-a9a20ec1b30a6802311efb8fe97bdc12f5ba554cf36f09501a3679fa7e70dc6bR729
And use `fmt` to build the full test name below.

@rossf7 WDYT? Or is our current go style guide to harcode the case number?

Reply:
It would be nicer to use the index.

> Or is our current go style guide to hardcode the case number?

I checked fmt and it uses the same approach as here. If we want to change this I think we should update the docs too.

https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md